monitor.cpp

```diff
@@ -3,36 +3,41 @@
 #include <thread>
 #include <chrono>
 #include <iostream>
-using std::cout, std::flush, std::this_thread::sleep_for, std::chrono::seconds;
+#include <string>
+using std::cout, std::flush, std::this_thread::sleep_for, std::chrono::seconds, std::string;
 
-int vitalsOk(float temperature, float pulseRate, float spo2) {
-  if (temperature > 102 || temperature < 95) {
-    cout << "Temperature is critical!\n";
-    for (int i = 0; i < 6; i++) {
-      cout << "\r* " << flush;
-      sleep_for(seconds(1));
-      cout << "\r *" << flush;
-      sleep_for(seconds(1));
-    }
-    return 0;
-  } else if (pulseRate < 60 || pulseRate > 100) {
-    cout << "Pulse Rate is out of range!\n";
+bool isNormalTemparature(float temperature) {
+    return ((temperature <= 102 && temperature >= 95));
+}
+
+bool isNormalPulseRate(float pulseRate) {
```

Comment:
Though the tool doesn't point it out, the range-comparison is repeated in temperature as well.
When something needs to be done in range comparison (like `<` instead of `<=`), it will need to be done in multiple places. Try to isolate range-comarison into its own function